### ISSUE TITLE
[Adjust Rule] SIM113: Take care of "continue"

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#32]
## Desired change

* **Rule(s)**: SIM113
* **Adjustment**: False-positive: Do not trigger if continue changes the number

## Example

```python
even_numbers = 0
for el in range(100):
    if el % 2 == 1:
        continue
    even_numbers += 1
```

**Comment by MartinThoma** [comment]
```
$ astpretty --no-show-offsets /dev/stdin <<< `cat example.txt`
Module(
    body=[
        Assign(
            targets=[Name(id='even_numbers', ctx=Store())],
            value=Constant(value=0, kind=None),
            type_comment=None,
        ),
        For(
            target=Name(id='el', ctx=Store()),
            iter=Call(
                func=Name(id='range', ctx=Load()),
                args=[Constant(value=100, kind=None)],
                keywords=[],
            ),
            body=[
                If(
                    test=Compare(
                        left=BinOp(
                            left=Name(id='el', ctx=Load()),
                            op=Mod(),
                            right=Constant(value=2, kind=None),
                        ),
                        ops=[Eq()],
                        comparators=[Constant(value=1, kind=None)],
                    ),
                    body=[Continue()],
                    orelse=[],
                ),
                AugAssign(
                    target=Name(id='even_numbers', ctx=Store()),
                    op=Add(),
                    value=Constant(value=1, kind=None),
                ),
            ],
            orelse=[],
            type_comment=None,
        ),
    ],
    type_ignores=[],
)
```
